src/app.js

```diff
@@ -0,0 +1,49 @@
+import axios from "axios";
+
+const localCurrency = (currencies) => {
+    let local = "";
+    for (let i = 0; i < currencies.length; i++) {
+        if (i>0) {
+            local += ' and ' + currencies[i].name + 's.';
+        }
+        else {
+            local= currencies[i].name;
+        }
+    }
+    console.log(local);
+    return local;
+}
+
+async function fetchCountryInformation() {
+    const BASE = "https://restcountries.com/";
+    const EndPoint = "v2/name/";
```

Comment:
Variabelnamen in camelCase schrijven
```suggestion
    const base = "https://restcountries.com/";
    const endPoint = "v2/name/";
```